spanner/api/SpannerTest/Tests.cs

```diff
@@ -749,9 +748,9 @@ void TestGetBackups()
             Assert.Equal(0, output.ExitCode);
             // BackupId should be a result of each of the 7 ListBackups calls and

```

Comment:
Does this depends on the order the tests are run? If it's so maybe don't change on this PR but definetely let's create an issue to fix this.

Reply:
It shouldn't depend on the order. It's just that there are two ListBackups calls that filter on the create_time and expire_time. Because we decided to reuse an existing backup rather than create a new one each time the test runs, the create_time and expire_time are no longer predictable to test for. This is why the check is for less than 7.